examples/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+cmake_minimum_required(VERSION 3.1)
+
+# Fetching version from header file
+file(STRINGS ../imgui.h ImGui_VERSION_NUM_HEADER_STRING
+    REGEX "#define[ \t]+IMGUI_VERSION_NUM[ \t]+([0-9]+)"
+    LIMIT_COUNT 1)
+string(REGEX REPLACE "#define[ \t]+IMGUI_VERSION_NUM[ \t]+([0-9]+)" "\\1"
+    IMGUI_VERSION_NUM "${ImGui_VERSION_NUM_HEADER_STRING}")
+math(EXPR IMGUI_VERSION_MAJOR "${IMGUI_VERSION_NUM} / 10000")
+math(EXPR IMGUI_VERSION_MINOR "(${IMGUI_VERSION_NUM} % 10000) / 100")
+math(EXPR IMGUI_VERSION_PATCH "${IMGUI_VERSION_NUM} % 100")
+
+project(imgui_examples
+    VERSION "${IMGUI_VERSION_MAJOR}.${IMGUI_VERSION_MINOR}.${IMGUI_VERSION_PATCH}"
+    LANGUAGES CXX)
+
+get_filename_component(ImGui_SRCDIR "${CMAKE_CURRENT_LIST_DIR}" DIRECTORY)
+
+include("${CMAKE_CURRENT_LIST_DIR}/ImGuiModule.cmake")
+
+set(ImGui_OPTIONS)
+
+imgui_option(EXAMPLES "Dear ImGui example applications" ON)
+imgui_option(BINDINGS "Dear ImGui example bindings" ON)
+imgui_option(MISC     "Dear ImGui misc features" ON)
+imgui_option(3RDPARTY "Dear ImGui example dependencies" ON)
+if(EMSCRIPTEN)
+    set(ImGui_OPENGL_LOADER_DEFAULT "GLEW")
+else()
+    set(ImGui_OPENGL_LOADER_DEFAULT "GL3W")
+endif()
+imgui_option(OPENGL_LOADER
+    "Dear ImGui OpenGL loader (GL3W, GLEW, GLAD or custom header)"
+    "${ImGui_OPENGL_LOADER_DEFAULT}"
+    STRINGS "GL3W" "GLEW" "GLAD")
+imgui_option(TOOLS "Dear ImGui auxiliary applications" OFF)
+imgui_option(PACKAGE "Dear ImGui packaging" OFF)
+
+file(GENERATE OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/ImGuiOptions.cmake"
+    CONTENT "${ImGui_OPTIONS_CMAKE}")
+
+include("${CMAKE_CURRENT_LIST_DIR}/ImGuiTargets.cmake")
+
+set(ImGui_DIR "${CMAKE_CURRENT_LIST_DIR}")
+
+imgui_example(example_null
+    TARGETS Core)
+
+if(EMSCRIPTEN)
+    imgui_example(example_emscripten
+        BINDINGS ImplSDL2 ImplOpenGL3)
+else()
+    imgui_example(example_glut_opengl2
+        BINDINGS ImplGLUT ImplOpenGL2)
+
+    imgui_example(example_sdl_opengl2
+        BINDINGS ImplSDL2 ImplOpenGL2)
+
+    imgui_example(example_sdl_opengl3
+        BINDINGS ImplSDL2 ImplOpenGL3)
+
+    imgui_example(example_sdl_vulkan
+        BINDINGS ImplSDL2 ImplVulkan)
+
+    imgui_example(example_glfw_opengl2
+        BINDINGS ImplGlfw ImplOpenGL2)
+
+    imgui_example(example_glfw_opengl2_freetype
+        BINDINGS ImplGlfw ImplOpenGL2
+        MISC FreeType)
+
+    imgui_example(example_glfw_opengl3
+        BINDINGS ImplGlfw ImplOpenGL3)
+
+    imgui_example(example_glfw_vulkan
+        BINDINGS ImplGlfw ImplVulkan)
+endif()
```

Comment:
Just out of curiosity, is emscripten exclusive of SDL/GLFW/etc? I thought SDL worked with emscripten.